third_party/sl4a/gradle.properties

```diff
@@ -0,0 +1,6 @@
+# This version code implements the versioning recommendations in:
+# https://blog.jayway.com/2015/03/11/automatic-versioncode-generation-in-android-gradle/
+VERSION_CODE=1000100
+VERSION_NAME=1.0.1-SNAPSHOT
```

Comment:
There were two commits in this PR: add the jcenter stuff necessary to release 1.0.0, and then bump the version to 1.0.1-SNAPSHOT. (I wasn't going to squash them).
I'll send a separate PR for the version bump to avoid confusion. Removed.